resources/templates/default.yml.j2

```diff
@@ -46,7 +46,7 @@ repos:
     ref: 71800396c2a07202265a8f92c85c33845a369494  # TODO: tag release
   etiss:
     url: "https://github.com/tum-ei-eda/etiss.git"
-    ref: 40c0b920cf4072adf189211ca44291534355dcfd
+    ref: db52781b01421491d01efec5dab8e81859c6afe9
```

Comment:
This will lead to termination issues in etiss_pulpino. Needs further investigation.